File: OpenGlTest/app/src/main/java/com/android/toutum/opengltest/MainActivity.java
package com.android.toutum.opengltest;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        // as the ContentView for this Activity.
        mGLView = new MyGLSurfaceView(this);
        setContentView(mGLView);
    }
}

File: ArchitectureComponentTest/app/src/main/java/com/eight25media/architecturecomponenttest/MainActivity.kt
package com.eight25media.architecturecomponenttest

import android.arch.lifecycle.Observer
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.eight25media.architecturecomponenttest.testlifecycle.LiveDataTest
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        LiveDataTest().observe(this, Observer { t ->
            run {
                text.text = t
            }
        })
    }
}
